content/en/docs/ambient/install/platform-prerequisites/index.md

```diff
@@ -79,199 +72,150 @@ There is an [open issue on the VPC CNI component](https://github.com/aws/amazon-
-    $ istioctl install --set profile=openshift-ambient --skip-confirmation
-    {{< /text >}}
+```bash
+helm install istio-cni istio/cni -n kube-system --set profile=ambient --set global.platform=openshift --wait
+helm install ztunnel istio/ztunnel -n kube-system --set profile=ambient --set global.platform=openshift --wait
+```
 
-{{< /tab >}}
+#### istioctl
 
-{{< /tabset >}}
+```bash
+istioctl install --set profile=openshift-ambient --skip-confirmation
+```
 
 ## CNI plugins
 
-The following configurations apply to all platforms, when certain {{< gloss "CNI" >}}CNI plugins{{< /gloss >}} are used:
+The following configurations apply to all platforms, when certain CNI plugins are used:
 
 ### Cilium
 
 1. Cilium currently defaults to proactively deleting other CNI plugins and their config, and must be configured with
-`cni.exclusive = false` to properly support chaining. See [the Cilium documentation](https://docs.cilium.io/en/stable/helm-reference/) for more details.
-1. Cilium's BPF masquerading is currently disabled by default, and has issues with Istio's use of link-local IPs for Kubernetes health checking. Enabling BPF masquerading via `bpf.masquerade=true` is not currently supported, and results in non-functional pod health checks in Istio ambient. Cilium's default iptables masquerading implementation should continue to function correctly.
-1. Due to how Cilium manages node identity and internally allow-lists node-level health probes to pods,
-applying any default-DENY `NetworkPolicy` in a Cilium CNI install underlying Istio in ambient mode will cause `kubelet` health probes (which are by-default silently exempted from all policy enforcement by Cilium) to be blocked. This is because Istio uses a link-local SNAT address for kubelet health probes, which Cilium is not aware of, and Cilium does not have an option to exempt link-local addresses from policy enforcement.
+   `cni.exclusive = false` to properly support chaining. See [the Cilium documentation](https://docs.cilium.io/en/stable/helm-reference/) for more details.
+2. Cilium's BPF masquerading is currently disabled by default, and has issues with Istio's use of link-local IPs for Kubernetes health checking. Enabling BPF masquerading via `bpf.masquerade=true` is not currently supported, and results in non-functional pod health checks in Istio ambient. Cilium's default iptables masquerading implementation should continue to function correctly.
+3. Due to how Cilium manages node identity and internally allow-lists node-level health probes to pods,
+   applying any default-DENY `NetworkPolicy` in a Cilium CNI install underlying Istio in ambient mode will cause `kubelet` health probes (which are by-default silently exempted from all policy enforcement by Cilium) to be blocked. This is because Istio uses a link-local SNAT address for kubelet health probes, which Cilium is not aware of, and Cilium does not have an option to exempt link-local addresses from policy enforcement.
```

Comment:
This is where you're getting your ordered number linting failure. Not sure why these are being changed, however?